gen/definitions/interface_port_channel.yaml

```diff
@@ -0,0 +1,40 @@
+---
+name: Interface Port_Channel
```

Comment:
Let's maybe just remove the underscore character here as this name should not contain any special characters.

Reply:
Done.